handsontable/src/core.js

```diff
@@ -3881,15 +3882,17 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
         case 'function':
           height = height(row);
           break;
+
         default:
           break;
       }
+
       if (typeof height === 'string') {
         height = parseInt(height, 10);
       }
     }
 
-    return height;
+    return (height && height < defaultRowHeight) ? defaultRowHeight : height;
```

Comment:
Can I ask you to cover the change with tests (`handsontable/test/e2e/core/getRowHeight.spec.js`)?  Also, I have found a bug ... when I declare `rowHeights: 0`, then the method returns `0` instead of the default theme row size.